src/com/lingoport/demo/java/ExamplePanel.java

```diff
@@ -165,7 +166,7 @@ public void actionPerformed(ActionEvent e) {
      */
     private void createDatePanel() {
 
-        datePanel_ = new JPanel(new GridBagLayout());
+        datePanel2_ = new JPanel(new GridBagLayout());
         
 //        String message = MessageFormat.format("{0} for Today: ", new Object[]{I18nUtils.getLocale()});
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9010/coding_rules#rule_key=squid%3ACommentedOutCodeLine)